modules/claims_api/lib/claims_api/error/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module ClaimsApi
+  module Error
+    module ErrorHandler
+      def self.included(clazz)
+        clazz.class_eval do
+          rescue_from ::Common::Exceptions::TokenValidationError, with: :token_validation_error
+        end
+      end
+
+      private
+
+      def token_validation_error
+        json = {
+          errors: [
+            {
+              title: 'Token Validation Error',
+              detail: 'Invalid token.',
+              code: '401',
```

Comment:
I think for now at least, we should not be returning a `code` attribute.
I believe this is sort of like the "sub-code" stuff that Kayla mentioned.  
But we don't yet have a system of defined sub-codes or how to correlate them to meaningful details.
We also haven't observed a clear need for a sub-coding system yet.
So for now, I say we just don't provide a `code` attribute.
We can always add things to the structure later, but we can never remove something once it exists.  So let's not add anything that we aren't 100% certain is needed.

Reply:
I agree that we don't need a `code` field (or `status` for that matter), but that's what we're returning right now, can we remove it, or is a client using it?

Reply:
So this stuff you're building is only going to be used in V2 (and versions beyond that) and there is no one currently consuming V2.  So we're free to build whatev we want here without fear of impacting current consumers.

Reply:
yea, depending on the library (assuming JS) theoretically `response.body.errors[0].code == response.body.errors[0].status == response.status_code` I feel like `code` should be reserved for a `subcodes` feature down the road Kayla mentioned.